### PR TITLE
[ZT] Create a tutorial for FastAPI and Cloudflare Zero Trust

### PR DESCRIPTION
**Pull request by eblaauw**

### CODE REVIEW

**Comment by oba11** [comment]
@eblaauw thank you for opening this PR, it will be really beneficial to alot of people 🚀

**Comment by kennyj42** [comment]
This looks great, thank you @eblaauw 

@ranbel my only thought is if we should link to: https://developers.cloudflare.com/cloudflare-one/identity/authorization-cookie/validating-json/ in the tutorial somewhere.

**Review by kennyj42 — APPROVED** [review]

File: content/cloudflare-one/tutorials/fastapi.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+This tutorial covers how to connect Cloudflare and Fast API.
+
+**Time to complete:** 15 minutes
+
+## 1. Create a validator
+
+1. Create a new file in your Fast API to validate the request, with the following code:
+```python
+TOKEN = "XXXXXX"
```
Comment (ranbel) [inline content/cloudflare-one/tutorials/fastapi.md]: It looks like TOKEN is not referenced anywhere else in the code. Can it be removed?

Comment (eblaauw) [reply content/cloudflare-one/tutorials/fastapi.md]: You're right, thank you. I have removed this.

File: content/cloudflare-one/tutorials/fastapi.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            return responses.return_403("Error decoding token")
+    if not valid_token:
+        return responses.return_403("Invalid token")
+
+    return True
+
+```
+
+## 2. Use the Fast API depedancy to validate the JWT
+```python
```
Comment (ranbel) [inline content/cloudflare-one/tutorials/fastapi.md:81]: Hi, I'm in the process of copy-editing this tutorial and Step 2 could use some clarification.
- Where does `router` come from?
- How does this code snippet fit into the broader context of the FastAPI application?

Comment (eblaauw) [reply content/cloudflare-one/tutorials/fastapi.md:81]: Hi, thank you for your comment. I understand that it was a bit confusing. I have added the imports in the tutorial now. (https://fastapi.tiangolo.com/tutorial/bigger-applications/#another-module-with-apirouter). Let me know if that helps.

Comment (ranbel) [reply content/cloudflare-one/tutorials/fastapi.md:81]: Yes thanks for updating the example! This tutorial has been edited and merged in #9330.

**Pull request closed without merging**